RestCommClient/Classes/RCConnection.m

```diff
@@ -300,9 +305,9 @@ - (void)sipManagerDidReceiveOutgoingDeclined:(SipManager*)sipManager;
         self.callingPlayer.currentTime = 0.0;
     }
     self.state = RCConnectionStateDisconnected;
-    [self.delegate connectionDidGetDeclined:self];
```

Comment:
@ognjenns actually removing these 3 lines and deferring the connectionDidGetDeclined event for when the busy tone finishes playing is problematic. We want the SDK API user to be notified that the connection is declined right when sipManagerDidReceiveOutgoingDeclined, not later, so that the event delivered to the user is properly synchronized with the call being declined.

Can you think of alternative ways to implement that to avoid this situation?

Reply:
Ok, will move it. We should put comment so we know in the future why we did it.